### PR TITLE
added html-css-popup task

### PR DESCRIPTION
**Pull request by veronkrot**
### HTML-CSS-Popup

[Demo](https://veronkrot.github.io/HTML-CSS-Popup/) |
[Code base](https://github.com/veronkrot/HTML-CSS-Popup)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/veronkrot/styles.css [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    cursor: pointer;
+}
+
+.apps-checkbox,
+.more-apps-checkbox,
+.disabled-item,
+.dividing-line,
+.more-from-kottans,
+.more-apps-checkbox:checked ~ .more-apps-button {
+    display: none;
```
Comment (A-Ostrovnyy) [inline submissions/veronkrot/styles.css:49]: Try to hide checkbox in right way. [Video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s)

File: submissions/veronkrot/styles.css [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    width: 100%;
+    height: 1px;
+    background-color: #7f8485;
+    margin: 10px 0 10px 0;
+}
+
+.list-item,
+.more-apps-checkbox:checked ~ .disabled-item,
+.more-apps-checkbox:checked ~ .more-from-kottans {
+    display: grid;
```
Comment (A-Ostrovnyy) [inline submissions/veronkrot/styles.css:85]: `display: grid` for positioning two elements in a column it's overkill if I understand you correctly.) In your situation will be enough to add `display: block; text-align: center` to the `.app-name`.

File: submissions/veronkrot/styles.css (hunk 1 above)
Comment (veronkrot) [reply submissions/veronkrot/styles.css:62]: fixed

File: submissions/veronkrot/styles.css (hunk 2 above)
Comment (veronkrot) [reply submissions/veronkrot/styles.css]: fixed

**Comment by A-Ostrovnyy** [comment]
Thanks for your submission. 
One last moment: add style on `:focus` state to the interactive elements: [Video](https://share.vidyard.com/watch/nV4EZQqeLsb8SU73NMSjKi?)
For future PR's if you fix some comment mark this comment resolved:
image

after adding your changes just click button `Re-request`: )
image

**Comment by veronkrot** [comment]
thanks for review)

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**